src/components/PupilJourneyComponents/QuizShortAnswer/QuizShortAnswer.test.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import "@testing-library/jest-dom/extend-expect";
+import "@testing-library/jest-dom";
+import { act, fireEvent } from "@testing-library/react";
+import { OakThemeProvider, oakDefaultTheme } from "@oak-academy/oak-components";
+
+import { QuizShortAnswer } from "./QuizShortAnswer";
+
+import {
+  QuizEngineContextType,
+  QuizEngineContext,
+} from "@/components/PupilJourneyComponents/QuizEngineProvider";
+import renderWithTheme from "@/__tests__/__helpers__/renderWithTheme";
+import { quizQuestions } from "@/node-lib/curriculum-api-2023/fixtures/quizElements.fixture";
+
+const shortAnswerQuestion = quizQuestions?.find(
+  (q) => q.answers?.["short-answer"] && q.answers?.["short-answer"].length > 0,
```

Comment:
Another potential use for those discriminators I mentioned! `isShortAnswer` 

```typescript
const shortAnswerQuestion = quizQuestions?.find((q) => isShortAnswer(q.answers));
```

Reply:
Okay. You've convinced me.  Lets work out a ticket where we can slot this stuff in.